### PR TITLE
Another fix for overlay check and for flags parsing

### PR DESCRIPTION
**Pull request by krnowak**
Commit messages say all.

### CODE REVIEW

File: pkg/bootstrap/util.go [hunk 1]
```diff
@@ -41,6 +41,10 @@ func PathSupportsOverlay(path string) error {
 		return fmt.Errorf("overlayfs is not available")
 	}
 
+	if err := os.MkdirAll(path, 0644); err != nil {
```
Comment (dongsupark) [inline pkg/bootstrap/util.go]: Do you probably mean `0755`?

Comment (krnowak) [reply pkg/bootstrap/util.go]: Ugh, yes. Will fix it.

**Comment by krnowak** [comment]
PR updated.

**Comment by dongsupark** [comment]
Looks good. Thanks!

**Pull request merged**